front/temporal/labs/activities.ts

```diff
@@ -211,6 +227,20 @@ export async function processGoogleDriveTranscriptActivity(
       `Could not find workspace for user (workspaceId: ${workspaceId}).`
     );
   }
+
+  const user_workspace_membership =
```

Comment:
camelCase should be used for variables.

(I thought we had a linter rule for that)